docs/contributing.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+## Setup
+This project uses an assortment of tools for the development.
+Currently included are:
+- [fvm](https://pub.dev/packages/fvm) for managing Flutter versions
+- [melos](https://pub.dev/packages/melos) for managing packages in this monorepo
+- [husky](https://pub.dev/packages/husky) for managing git hooks
+- [commitlint_cli](https://pub.dev/packages/commitlint_cli) for validating commits according to the conventional commits specification
+
+To set up all these tools run the `./tool/setup.sh` script.
+Note that you need to have Dart installed and `~/.pub-cache/` needs to be in your PATH before running the script.
```

Comment:
`.pub-cache/bin` as you only want to add packages explicitly activated with `dart pub global activate`

Maybe a link to https://dart.dev/tools/pub/cmd/pub-global#running-a-script-from-your-path might be good as it also describes the craziness of windows (`%LOCALAPPDATA%\Pub\Cache\bin`)